source/extensions/extensions_build_config.bzl

```diff
@@ -106,6 +106,12 @@ EXTENSIONS = {
 
     "envoy.filters.udp_listener.udp_proxy":             "//source/extensions/filters/udp/udp_proxy:config",
 
+    #
+    # Request ID utilities
+    #
+    
+    "envoy.request_id_utils.uuid":                      "//source/extensions/request_id_utils/uuid:config",
```

Comment:
We shouldn't need to register this. I would make sure this is compiled in in all cases per other comments so we don't break users in confusing ways.